### PR TITLE
golang guide with huma

### PR DESCRIPTION
**Pull request by philsturgeon**
In the same spirit as the Python FastAPI guide, this Golang Huma guide is pretty much the exact same thing. The tool is based on FastAPI so its an incredibly similar guide.

### CODE REVIEW

File: src/_guides/bump-sh-tutorials/go.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Deploying docs from FastAPI
+authors: phil
+excerpt: Generating OpenAPI for Go APIs
+---
+
+[Huma](https://fastapi.tiangolo.com/) is a Golang framework that allows you to create APIs, and automatically generates their documentation, based on the OpenAPI specification.
```
Comment (ChristopheDujarric) [inline src/_guides/bump-sh-tutorials/go.md:7]:
```suggestion
[Huma](https://huma.rocks/) is a Golang framework that allows you to create APIs, and automatically generates their documentation, based on the OpenAPI specification.
```

File: src/_guides/bump-sh-tutorials/go.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Deploying docs from FastAPI
```
Comment (ChristopheDujarric) [inline src/_guides/bump-sh-tutorials/go.md:2]:
```suggestion
title: Deploying docs from Huma
```

Comment (ChristopheDujarric) [reply src/_guides/bump-sh-tutorials/go.md:2]: I'd also rename the file to `src/_guides/bump-sh-tutorials/huma.md`

File: src/_guides/bump-sh-tutorials/go.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+4. Deploy your doc to Bump.sh with
+
+```bash
+bump deploy http://127.0.0.1:8888/openapi.yaml \
+  --doc my-documentation-name \
+  --token my-documentation-token
+```
+
+If for any reason you are unable to download that `openapi.yaml` from a URL despite being able to access it, it;s likely some odd network proxy settings on your machine or wifi. Grab it directly with wget or similar.
```
Comment (ChristopheDujarric) [inline src/_guides/bump-sh-tutorials/go.md:37]:
```suggestion
If for any reason you are unable to download that `openapi.yaml` from a URL despite being able to access it, it's likely some odd network proxy settings on your machine or wifi. Grab it directly with wget or similar.
```

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]

**Comment by philsturgeon** [comment]
Sorry that was sloppy. Merged all that, and will get focused on the bigger and better guides suggested.

**Review by ChristopheDujarric — APPROVED** [review]

**Pull request merged**